web/Weblate-server.qmd

```diff
@@ -238,21 +238,27 @@ Weblate will provide a suggestion on how to fix, but in short:
     git svn rebase
     ```
 
-6. Optionally, if not yet done, install [helpers](https://github.com/JulienPalard/po3way) to make it easier to deal with the git conflicts (inside Docker):
+6. Optionally, if not yet done, install [helpers](https://github.com/JulienPalard/po3way) to make it easier to deal with the git conflicts (inside Docker). Start Docker as root to install `po3way` system-wide:
+
 
    ```sh
-   pip install po3way
+   $ sudo docker exec -u root -ti weblate-docker_weblate_1 bash
    ```
 
-   Then add this folder to `$PATH` or symlink it, e.g. starting Docker as root:
+   Then in the Docker container:
+
+   ```sh
```

Comment:
I can't quite tell whether this is recognized as a syntax & we don't get a preview here. Does Weblate support syntax highlighting anyway?

GitHub may or may not, this doc:

https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting

Says we should look at this yaml config to see what's valid:

https://github.com/github-linguist/linguist/blob/559a6426942abcae16b6d6b328147476432bf6cb/lib/linguist/languages.yml#L282-L283

But I don't know if we're looking at `ace_mode` or `codemirror_mode` as the source of valid abbreviations.

Personally I'm accustomed to use ```` ```shell ````.

Reply:
Tried this out on GitHub wiki, seems `sh` and `shell` are equivalent, at least for some trivial commands:

![Screenshot from 2024-02-25 00-05-44](https://github.com/r-devel/translations/assets/7606389/4c00a44e-4002-47a4-9c88-e236db51f1d4)

That's good enough for me.

Reply:
>  Weblate support syntax highlighting anyway?

I'm not sure what you mean .. this is to update the Quarto doc.

> I can't quite tell whether this is recognized as a syntax & we don't get a preview here. 

yeah, I've used both `sh` and `shell` in the past quite a lot, should work.